consensus/ibft/timeout_provider.go

```diff
@@ -0,0 +1,28 @@
+package ibft
+
+import (
+	"math"
+	"time"
+)
+
+const (
+	baseTimeout = 10 * time.Second
+	maxTimeout  = 300 * time.Second
+)
+
+// exponentialTimeout calculates the timeout duration in seconds as exponential function
+// where maximum value returned can't exceed 300 seconds
+// t = 10 + 2^exponent	where exponent > 0
+// t = 10				where exponent = 0
```

Comment:
The second condition looks shifted

Reply:
not sure why it looks like this on GH, in IDE looks aligned with the "where" from the line above

Reply:
can confirm that GitHub on Firefox indents wrong. @Kourin1996 GoLand has GitHub PRs integrated, it's never too late to move from VSCode :)